alembic/versions/a0e48c400be4_store_submit_time_of_srpm_builds.py

```diff
@@ -0,0 +1,66 @@
+"""Store submit time of SRPM builds
+
+Revision ID: a0e48c400be4
+Revises: 521433a1fdb0
+Create Date: 2020-09-01 06:52:19.289499
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "a0e48c400be4"
+down_revision = "521433a1fdb0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("celery_tasksetmeta")
+    op.drop_table("celery_taskmeta")
```

Comment:
:eyes: 

is this something we really want?

Reply:
Dont know why this happened. Will remove all of this form the autogenerated script